generation/2d_ldm/2d_ldm_tutorial.ipynb

```diff
@@ -0,0 +1,1526 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    max_epochs = 100
</pre><p>Let's comment this one and add a small max_epochs here to speed up the tests and daily CI runs. What do you think?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>I thought this was automatically replaced by runner.sh? </p>

Reply:
This was something I thought runner.sh was going to overwrite.

Reply:
Got it.